web/src/components/gitops/GitOpsRepoDetails.jsx

```diff
@@ -270,7 +270,13 @@ class GitOpsRepoDetails extends React.Component {
 
             <div className="flex">
               {this.props.showCancelBtn && <button className="btn secondary dustyGray u-marginRight--10" type="button" onClick={this.props.onCancel}>Cancel</button>}
-              <button className={`btn primary blue ${(finishingSetup || !this.allowUpdate()) && "is-disabled"}`} type="button" onClick={this.onFinishSetup}>{finishingSetup ? "Finishing setup" : "Finish GitOps setup"}</button>
+              <button
+                className={`btn primary blue ${(finishingSetup || !this.allowUpdate()) && "is-disabled"}`}
+                type="button"
+                onClick={(finishingSetup || !this.allowUpdate()) ? null : this.onFinishSetup}
```

Comment:
is there a reason we don't just add this attribute? `disabled={finishingSetup || !this.allowUpdate()}`

Reply:
oh my bad. nope, no reason, updating!

Reply:
👍 thanks